src/index.ts

```diff
@@ -1,61 +1,148 @@
-import { Reshuffle, BaseConnector, EventConfiguration } from 'reshuffle-base-connector'
+import {
+  BaseHttpConnector,
+  EventConfiguration,
+  Reshuffle,
+} from '@reshuffle/base-connector'
+import {
+  parseFilter,
+  unwrapDates,
+  wrapDates,
+} from "./util"
 
-export interface _CONNECTOR_NAME_ConnectorConfigOptions {
-  var1: string
-  // ...
+import {Request, Response} from 'express'
```

Comment:
the formatting is wrong.
`npm lint` will fix that.
Also turn on circleci which will run the checks (see instructions in template readme)